src/oca_github_bot/tasks/migration_issue_bot.py

```diff
@@ -84,7 +84,8 @@ def _set_lines_issue(gh_pr_user_login, gh_pr_number, issue_body, module):
 
 
 def _mark_migration_done_in_migration_issue(gh_repo, target_branch, gh_pr):
-    migration_issue = _find_issue(gh_repo, target_branch)
+    milestone = _create_or_find_branch_milestone(gh_repo, target_branch)
```

Comment:
I think this shouldn't create the milestone.

Reply:
I guess it does no harm in practice. Merging to unblock the situation.